src/main/java/com/sda/Warehouse/models/User.java

```diff
@@ -0,0 +1,46 @@
+package com.sda.Warehouse.models;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.validator.constraints.Email;
+
+import javax.persistence.*;
+import javax.validation.constraints.Size;
+
+@Entity(name = "users")
+@Data
```

Comment:
Why this model is mutable? Do you need Setters?

Reply:
Ok, I will change it and I will add setter only for field isActive